common/src/chain/config/mod.rs

```diff
@@ -215,6 +215,18 @@ impl ChainConfig {
         self.is_last_block_in_epoch(height) && current_epoch_index >= sealed_epoch_distance_from_tip
     }
 
+    pub fn sealed_epoch_index(&self, height: &BlockHeight) -> Option<EpochIndex> {
```

Comment:
Please write tests for this function. The arithmetic can easily be done wrong with bad consequences.

Reply:
I'll do these tests when I add target calculation for PoS